puppet/data/osfamily/RedHat-9.yaml

```diff
@@ -1,3 +1,2 @@
 ---
 redmine::repo_branch: 'new-redmine'
```

Comment:
While you're at it: remove this and keep the old deployment on some branch like `old-redmine`?

Reply:
yeah…

Reply:
https://github.com/theforeman/redmine/pull/32